### ISSUE TITLE
Visualisation does not wot work because np.mat is removed

### ISSUE THREAD

**Issue by weidtn** [espressomd/espresso#4979]
Whenever I try to run an example that uses the visualisation I get errors, because numpy's mat was removed in numpy 2.0:

```
AttributeError: `np.mat` was removed in the NumPy 2.0 release. Use `np.asmatrix` instead.. Did you mean: '_mat'?                                                                                                                    
```

**Comment by jngrad** [comment]
This was already fixed by #4971.

Also, please keep in mind we don't officially support NumPy 2.0 yet:
https://github.com/espressomd/espresso/blob/8c90c3d1e485a2c3107fc22eb24e037b8d0cdf15/requirements.txt#L6

**Comment by weidtn** [comment]
Oh okay sorry, did not see that when I was searching github. Maybe because it was not an issue.
